protocols/virtualfund/virtualfund_test.go

```diff
@@ -63,7 +63,7 @@ func TestMarshalJSON(t *testing.T) {
 	encodedVfo, err := json.Marshal(vfo)
 
 	if err != nil {
-		t.Fatalf("error encoding direct-fund objective %v", vfo)
+		t.Fatalf("error encoding virtualfun objective %v", vfo)
```

Comment:
```suggestion
		t.Fatalf("error encoding virtualfund objective %v", vfo)
```

Reply:
I'm glad you are here to police the police.